<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1179</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="main_stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1161</width>
      <height>641</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>90</y>
        <width>281</width>
        <height>421</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="orentationS_verticalLayout">
       <item>
        <widget class="QPushButton" name="hetero_pushButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="ressources.qrc">
           <normaloff>:/logos/Hetero_icon.png</normaloff>:/logos/Hetero_icon.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="homo_pushButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="ressources.qrc">
           <normaloff>:/logos/homo_icon.png</normaloff>:/logos/homo_icon.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="lesbi_pushButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="ressources.qrc">
           <normaloff>:/logos/lesbian_icon.png</normaloff>:/logos/lesbian_icon.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
    <widget class="QWidget" name="page_3"/>
    <widget class="QWidget" name="page_4"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1179</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRetour"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRetour">
   <property name="text">
    <string>Retour</string>
   </property>
   <property name="toolTip">
    <string>Revenir au menu principal</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
